src/collections/blog/2019/2019-10-12-hacktoberfest-2019-is-heating-up/index.mdx

```diff
@@ -11,6 +11,9 @@ tags:
 published: true
 redirect_from:
  - /blog/hacktoberfest-2019-heating-up/
+ - /blog/landscape/hacktoberfest-2019-heating-up
+ - /blog/landscape/hacktoberfest-2019-heating-up/
+ - /blog/landscape/hacktoberfest-2019-heating-up.html
```

Comment:
Is the .html showing up in the 404 report?

Reply:
@leecalcote yes it is

Reply:
@iArchitSharma this PR is pending a more informed response. Where are the .html 404s coming from, specifically? A bot is making them up? Coming from the sitemap?